src/test/java/com/google/api/generator/engine/ast/JavaDocCommentTest.java

```diff
@@ -0,0 +1,69 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.api.generator.engine.ast;
+
+import static com.google.common.truth.Truth.assertThat;
+
+import java.util.Arrays;
+import java.util.List;
+import org.junit.Test;
+
+public class JavaDocCommentTest {
+  @Test
+  public void JavaDocComment_basic() {
+    String content = "this is a test comment";
+    JavaDocComment javaDocComment = JavaDocComment.builder().addComment(content).build();
```

Comment:
withComment?

Reply:
added withComment in javaDocComment for quick build. done.